packages/driver/src/cy/multidomain/index.ts

```diff
@@ -1,29 +1,62 @@
 import Bluebird from 'bluebird'
-import $Log from '../../cypress/log'
 import { createDeferred } from '../../util/deferred'
+import $utils from '../../cypress/utils'
+import $errUtils from '../../cypress/error_utils'
+
+export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy, state: Cypress.State, config: Cypress.InternalConfig) {
+  let timeoutId
 
-export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy) {
   // @ts-ignore
-  Cypress.on('cross:domain:html:received', () => {
+  const communicator = Cypress.multiDomainCommunicator
+
+  communicator.on('html:received', () => {
     // when a secondary domain is detected by the proxy, it holds it up
     // to provide time for the spec bridge to be set up. normally, the queue
     // will not continue until the page is stable, but this signals it to go
-    // ahead because we're anticipating multidomain
+    // ahead because we're anticipating multi-domain
     // @ts-ignore
     cy.isAnticipatingMultidomain(true)
```

Comment:
nit: Multidomain or MultiDomain?

```suggestion
    cy.isAnticipatingMultiDomain(true)
```

Reply:
This is getting fixed codebase-wide in #19601.